common/djangoapps/adaptive_learning/migrations/0002_courseadaptivelearningflag.py

```diff
@@ -2,9 +2,8 @@
 from __future__ import unicode_literals
 
 from django.db import migrations, models
-import django.db.models.deletion
-import openedx.core.djangoapps.xmodule_django.models
 from django.conf import settings
```

Comment:
@tomaszgy I think `django.conf` should precede `django.db`?

Reply:
@itsjeyd, I'll move it but why would it need to precede `django.db`? Does it need `settings` to be imported before we import something from `django.db`?

Reply:
@tomaszgy There's no special reason other than wanting imports to be ordered alphabetically (`django.c...` should precede `django.d...`).